salt/state.py

```diff
@@ -488,7 +488,7 @@ def verify_high(self, high):
                                         # I am going beyond 80 chars on
                                         # purpose, this is just too much
                                         # of a pain to deal with otherwise
-                                        if req_val in reqs:
+                                        if req_val in sorted(reqs):
```

Comment:
Perhaps `reqs` in line 389 should become an ordered dictionary. Would that make this predictable on both py2 and py3?